app/Resources/translations/default/Install.xlf

```diff
@@ -497,8 +497,8 @@ Comment: We fill the process step used for Ajax queries</note>
         <note>Line: 30</note>
       </trans-unit>
       <trans-unit id="74987401b3b33cc9542cd8585c08de98">
-        <source><![CDATA[To use PrestaShop, you must <a href="http://doc.prestashop.com/display/PS16/Installing+PrestaShop#InstallingPrestaShop-Creatingadatabaseforyourshop" target="_blank">create a database</a> to collect all of your store's data-related activities.]]></source>
-        <target><![CDATA[To use PrestaShop, you must <a href="http://doc.prestashop.com/display/PS16/Installing+PrestaShop#InstallingPrestaShop-Creatingadatabaseforyourshop" target="_blank">create a database</a> to collect all of your store's data-related activities.]]></target>
+        <source><![CDATA[To use PrestaShop, you must <a href="https://devdocs.prestashop.com/1.7/basics/installation/localhost/#creating-a-database-for-your-shop" target="_blank">create a database</a> to collect all of your store's data-related activities.]]></source>
```

Comment:
Indeed this link is old but I think the right link is rather https://doc.prestashop.com/display/PS17/Installing+PrestaShop#InstallingPrestaShop-Creatingadatabaseforyourshop

devdocs.prestashop.com is the developer documentation while doc.prestashop.com is the user (merchant) documentation. The installer aims for users I believe 🤔 (@LouiseBonnard please tell me if I got it wrong)